src/icon-tab-bar.scss

```diff
@@ -722,6 +753,9 @@ $fd-icon-tab-bar-semantic-values: (
   }
 
   &__list {
+    max-height: 20rem;
```

Comment:
Wondering if we want to leave this up to the application developer

Reply:
I added default height, because without height/overflow: auto. Popover is looked ugly, It is cut off and climbs out of the page if there are a lot of entities. You can check it to remove max-height